cpp/examples/interop/interop.cpp

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "arrow/array/array_binary.h"
+#include "arrow/type.h"
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/interop.hpp>
+#include <cudf/io/csv.hpp>
+
+#include <rmm/mr/device/device_memory_resource.hpp>
+
+// Helper functuons to create StringViews
+inline arrow::BinaryViewType::c_type ToInlineStringView(const void* data, int32_t size)
```

Comment:
All the function names should follow cudf convention.
```suggestion
inline arrow::BinaryViewType::c_type to_inline_string_view(const void* data, int32_t size)
```